region/brinstar/red/Alpha Power Bomb Room.json

```diff
@@ -208,6 +208,122 @@
+            "canTrickyJump",
+            "canHitbox"
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 3}},
+            "h_canArtificialMorphBombHorizontally",
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 1
+            }}
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 2}},
+            "h_canArtificialMorphBombHorizontally",
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 1
+            }},
+            "canTrickyJump",
+            "canHitbox"
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 2}},
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 2
+            }}
+          ]}
+        ]}
+      ],
+      "note": [
+        "Traverse the room with a small number of Power Bombs.",
+        "If the Chozo item is still there, it is best to exit G-Mode before the final Power Bomb goes off to ensure that PLMs aren't overloaded.",
+        "This strat can be done easily with 7 Power Bombs. Simply kill all of the Boyons before progressing.",
+        "The final Power Bomb needs to be placed on the second to last floating platform or later to break the wall.",
+        "Samus can use one Power Bomb and a Boyon hit to get from the door to the second platform, or from the second to fourth platform.",
+        "Alternatively, when next to the Boyons, place a Power Bomb while the Boyon is low, slightly after the peak of the jump, hitbox through the Boyons.",
+        "If the Boyons are killed, Samus can horizontally boost far enough to skip a platform.",
+        "Samus can cross the room with any of the following: 7 PB; 6 PB and a horizontal boost; 5 PB and hitboxing the first Boyons; 5 PB and damage from the first Boyons;",
+        "4 PB and damage from the first Boyons and hitboxing the second; 3 PB and hitting both sets of Boyons.",
+        "Note that if Samus falls into a Samus Eater, she will be teleported into it when exiting G-Mode."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "G-Mode Morph Power Bomb the Blocks (Thorn Damage Boost)",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "canIframeSpikeJump",
+        "canNeutralDamageBoost",
+        "canTrickyJump",
+        {"thornHits": 3},
+        "h_canArtificialMorphPowerBomb",
+        {"samusEaterFrames": 160}
+      ],
+      "note": [
+        "Traverse the room while rolling on the thorns and avoiding the Samus Eaters.",
+        "Roll from the first platform onto the first set of thorns. When just past the first Boyons, turn right to damage boost up and hit the block and fall back on the thorns.",
```

Comment:
Is this the spikes by the door?  You can roll over the first Vileplume from the door ledge.  2 thorn hits doesn't feel bad.

Reply:
this is the thorns under the boyons (skipping the first vileplume) I'll make that more clear. You can get into the third vileplume with only 2 spike hits without much difficulty?

Reply:
Right now I find flicking the morphball right as iframes end easier than stopping on the safe spot in the thorns.  Neither are something I could do every time.